01-node-tutorial/answers/writeWithPromisesAwait.js

```diff
@@ -0,0 +1,37 @@
+const { writeFile, readFile } = require("fs").promises;
+
+// writeFile returns promises
+
+const writer = async () => {
+	try {
+		await writeFile("./temp.txt", `Hello, I love you, `);
+		await writeFile("./temp.txt", `won't you tell me your name\n`, {
+			flag: "a",
+		});
+		await writeFile("./temporary/temp.txt", `Hello, I love you!`, {
```

Comment:
I think this should be `"./temp.txt"` like lines 7-8.

Reply:
oh my, you're right. I was confused slightly by where CTD wanted me to save temp.txt, either in ./temporary  folder (where it was created "last week") in /answers for a .gitignore practice. `Then create an async function called writer that takes 0 arguments, and that writes three lines to a file named temp.txt, by calling the writeFile function with await.`. I've forgotten to change all the paths. thanks.